src/applications/financial-status-report/components/SubmissionError.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const SubmissionError = () => {
+  return (
+    <div className="usa-alert usa-alert-error">
+      <div className="usa-alert-body">
+        <h4>We can’t submit your application right now</h4>
+        <p className="usa-alert-text">
+          We’re sorry. Something went wrong and we can't submit your
```

Comment:
Might not be in scope but on the `10-10CG` we have different content depending on the BE response just something to think about

Reply:
yep this is just a placeholder until we can iterate on it